src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,19 +1,78 @@
 # TO-DO: complete the help function below to merge 2 sorted arrays
+
```

Comment:
Great job solving this, write some comments, I am going to ask you about it during 1:1.

Those comments will help you explain easily unless you know it very well.